src/lib/third_party/src/ndpi_crc.c

```diff
@@ -0,0 +1,97 @@
+#include "ndpi_main.h"
```

Comment:
The same here. Could you move this code in `src/lib/ndpi_analyze.c`, near the crc32 stuff, please?